app/src/screens/SignIn/index.tsx

```diff
@@ -95,10 +99,16 @@ const SignIn: React.FC = () => {
   });
 
   const handleLogin = async () => {
+    navigation.reset({ routes: [{ name: 'Busy' }] });
+
     const loginAction = await dispatch(loginAndRegister());
     if (loginAndRegister.rejected.match(loginAction)) {
       console.error(loginAction.payload || loginAction.error);
+      navigation.reset({ routes: [{ name: 'Sign In' }] });
+      return;
     }
+
+    navigation.reset({ routes: [{ name: 'App', params: { screen: 'Home' } }] });
```

Comment:
on successful login why `reset` and not just `navigate` to the page?

Reply:
We don't want the user to be able to "press back" and to back to the login screen. I would rather just reset the navigation stack.